### ISSUE TITLE
Introspecting the state of pairing

### ISSUE THREAD

**Issue by jihoonl** [rosjava/android_remocons#36]
Even the app has started in the tablet, it is hard to now whether rapp in the robot has been started properly or not.

So does stopping.

https://github.com/turtlebot/turtlebot_android/issues/10

[Daniel] add general introspection to this idea.

**Comment by corot** [comment]
It´s no hard... it´s impossible! (afaik).

+1 to include a widget that periodically calls app manager status  service
and shows remote app´s  status, Can be added automatically to any app, as
now happens with battery status and back button

2013/10/25 Jihoon Lee notifications@github.com

**Comment by stonier** [comment]
Yeah, backing up Jorge's comment. Impossible right now and I've realised the need for this for a while. This is the same discussion as we had with the rqt dashboard. But directing effort to android is probably a better option because the netbook is usually closed.

It's a bit of work though, so not likely to happen immediately. Added it to the Todo milestone.

**Comment by stonier** [comment]
We also really don't have an easy way to know if an app has started properly or not.

https://github.com/robotics-in-concert/rocon_app_platform/issues/86
